app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -36,9 +41,39 @@ class ExplorationActivityTest {
     }
   }
 
+  @Test
+  fun textExplorationActivity_onBackPressed_showsStopExplorationDialog() {
```

Comment:
Rename this to `testExplorationActivity_onBackPressed_showsStopExplorationDialog()`
Ditto elsewhere

Reply:
Done. everywhere in this file.